cli/src/test/unlock.test.ts

```diff
@@ -0,0 +1,49 @@
+import anyTest, { TestInterface } from 'ava';
+import * as td from 'testdouble';
+import { Context, addFake } from './fake_lycan';
+import { success } from 'specshell';
+import { createApp } from './utils';
+import { UnauthorizedError, NotFoundError } from '@binaris/spice-koa-server/interfaces';
+
+const test = anyTest as TestInterface<Context>;
+
+addFake(test);
+
+const makeApp = createApp({locked: true, lockReason: 'template application'});
```

Comment:
```suggestion
const app = createApp({locked: true, lockReason: 'template application'});
```
AFAICS this is no longer a "`make`", it's the app itself.